src/rx.cpp

```diff
@@ -820,14 +827,15 @@ void Aggregator::send_packet(int ring_idx, int fragment_idx)
 
     if (packet_seq > seq + 1 && seq > 0)
     {
+        WFB_LOG(LogLevel::WARNING, "Lost packet %u\n", seq + 1);
```

Comment:
@vertexodessa Message should be:  `"Lost %u packets", (packet_seq - seq - 1)`
but this already tracked in `dump_stats`

```
if(count_p_override)
    {
        fprintf(stderr, "%u block overrides\n", count_p_override);
    }
    
    if(count_p_lost) 
    {
        fprintf(stderr, "%u packets lost\n", count_p_lost);
    }
```

Reply:
dump_stats is called within interval while this will be called immediately whe a packet was lost?
then I guess it makes sense to log this, though I'll change the log level to INFO

Reply:
ok. But in case of bad signal reception you will be flooded by these messages

Reply:
hm after another thought, a lost packet may be considered warning, so I'd prefer to leave it as is.
Should I change the default log level of the `StdErrLogger` to `ERROR` then? 
If I won't, this will be an additional log compared to the current implementation.

Reply:
Please convert it to INFO or DEBUG. Because it will flood stderr during normal wfb-ng operation

Reply:
sure, converted. I was considering this log moreimportant than stats, but actually it should be OK for them to have the same importance.